### PR TITLE
Add ability to configure server port

### LINKED ISSUE #729: Can we use process.env.PORT, too?
I use environments like c9.io and most of the time `9000` is not the right port for me....

My idea is to add support for `process.env.PORT`  ;-)

I will submit a PR asap as version `3.0.2`.

### PR DESCRIPTION
**Pull request by UlisesGascon**
Closed #729

### CODE REVIEW

**Review by silvenon — COMMENTED** [review]
Yeah, we should definitely have this configureable. I was thinking that we do this using parameters instead of environment variables.

```
gulp serve --port 8000
```

You can use yargs for this, see [this Stack Overflow answer](https://stackoverflow.com/questions/28538918/pass-parameter-to-gulp-task). With environment variables we would have to use cross-env, so it's an extra dependency either way.

What do you think?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-webapp",
-  "version": "3.0.1",
+  "version": "3.0.2",
```
Comment (silvenon) [inline package.json]: Don't bump the version, I will do it, instead just let me know that you would like to see this feature published ASAP. 😉

**Review by silvenon — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (UlisesGascon) [reply package.json]: Oh! Great! I will update it now ^^

**Comment by UlisesGascon** [comment]
Hi @silvenon! I added yargs to the project.
- `gulp serve`, `gulp serve:test` and `gulp serve:dist` now accept additional param `--port=xxxx`., by default we use 9000.
- I added some examples to the `Readme.md` too...

File: app/templates/gulpfile.js [hunk 2]
```diff
@@ -7,11 +7,13 @@ const wiredep = require('wiredep').stream;
 const runSequence = require('run-sequence');
 const autoprefixer = require('autoprefixer');
 const cssnano = require('cssnano');
+const argv = require('yargs').argv;
```
Comment (silvenon) [inline app/templates/gulpfile.js]: So sorry to nitpick, but you can destructure this instead:
```suggestion
const { argv } = require('yargs');
```

**Review by silvenon — APPROVED** [review]
Looks great, thanks for updating the docs too! After updating what I left in the comments, I'll merge.

File: app/templates/gulpfile.js (hunk 2 above)
Comment (UlisesGascon) [reply app/templates/gulpfile.js]: Right! Let me change that 👍

**Comment by UlisesGascon** [comment]
Awesome! First PR almost done 👍

**Pull request merged**